content/community/index.md

```diff
@@ -19,50 +19,31 @@ A vibrant community of contributors is the reason this project exists. The [Gala
 
 In 2010 the National Science Foundation initiated a yearly gathering of Galaxy users and developers by providing the project with a grant supplement. Since then this gathering has grown into an annual event known as the **[Galaxy Community Conference (GCC)](/gcc/)**. GCC alternates between North America and elsewhere, and provides a forum for sharing knowledge and building collaborations. There have been over 200 participants every year since 2012.
 
+# Interacting communities with Galaxy
```

Comment:
I am not sure about the wording for the title. 
What about "external communities interacting with Galaxy".